foundations/javascript_basics/installing_nodejs.md

```diff
@@ -25,7 +25,7 @@ To install nvm properly, you'll need `curl`. Simply run the command below:
 sudo apt install curl
 ~~~ 
   
-Note: You may need to update Ubuntu to the latest version for the Curl installation to complete. If so, run the command below:
+Note: You may need to update the package lists to the latest version for the Curl installation to complete. If so, run the command below:
```

Comment:
I would prefer we keep the ubuntu and phrase it as "the Ubuntu package lists". The reason being is the course doesn't touch on what the package lists of ubuntu are and it would be helpful to have it as "Ubuntu system package lists" so folks don't get confused regarding what the package lists are for.

Reply:
@yascont Do you still want to pursue this PR?

Reply:
@xandora do you mean merging it ? if so, yes

Reply:
@yascont There were some changes that @linkonsat requested. They'll need to be addressed before we can merge.